brave/browser-laptop
# Refactor about:passwords with Aphrodite

Closes #7335

- passwords.less is removed
- PasswordsTh, PasswordsTr (HeadTr), and PasswordsTd (ActionsTd) are introduced
- classes are replaced with data-test-id

Auditors:

Test Plan:
1. Remember a password by logging in GitHub
2. Open about:passwords
3. Make sure styling is not broken
